Projects/surface_photometry.ipynb

```diff
@@ -0,0 +1,1332 @@
+    "        and the filters in the image are used.\n",
+    "    stretch: int\n",
+    "        The linear stretch of the image.\n",
+    "    Q: int\n",
+    "        The Asinh softening parameter.\n",
+    "    scale: list of 3 floats, each less than 1. (default: None)\n",
+    "        Re-scales rgb channels  \n",
+    "        \n",
+    "    Returns\n",
+    "    -------\n",
+    "    rgb: ndarray \n",
+    "        RGB (integer, 8-bits per channel) colour image as an NxNx3 numpy array.\n",
+    "    \"\"\"\n",
+    "    \n",
+    "    # If the image only has 3 bands, reverse the order of the bands to produce the RGB image\n",
+    "    if len(image) == 3:\n",
+    "        bgr = image.filters\n",
+    "    # Extract the primary image component of each Exposure with the .image property, and use \n",
+    "    # .array to get a NumPy array view.\n",
+    "\n",
+    "    if scale is None:\n",
+    "        \n",
+    "        r_im = image[bgr[2]].array # numpy array for the r channel\n",
+    "        g_im = image[bgr[1]].array # numpy array for the g channel\n",
+    "        b_im = image[bgr[0]].array # numpy array for the b channel\n",
+    "    \n",
+    "    else:\n",
+    "        ### manually re-scaling the images here \n",
+    "        r_im = image[bgr[2]].array*scale[0]\n",
+    "        g_im = image[bgr[1]].array*scale[1]\n",
+    "        b_im = image[bgr[0]].array*scale[2]     \n",
+    "    \n",
+    "    rgb = make_lupton_rgb(image_r=r_im,  \n",
+    "                          image_g=g_im,  \n",
+    "                          image_b=b_im,  \n",
+    "                          stretch=stretch, Q=Q)  # parameters used to stretch and scale the pixel values\n",
+    "    \n",
+    "    return rgb          "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## **Part 1: Selecting a galaxy by coordinates**"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "As detailed in other tutorials, first we need to set up a Butler based on the parameters we obtained in the previous section.  We do this by galaxy coordinates, looping through available imaging repositories (DEEP which includes DEEP and ULTRA-DEEP tracts and WIDE). We selected this repository as it is the most complete dataset of HSC PDR2 in the LSP at the time of writing this notebook [(more info)]( https://confluence.lsstcorp.org/display/DM/S20+HSC+PDR2+Reprocessing). Thank you Yusra AlSayaad for the much needed help with this!\n",
+    "\n",
+    "NOTE: currently this is built for HSC imaging data only.  Also, the current notebook requires Butler gen. 2; this will malfunction with Butler gen. 3 due to the HSC repository we are accessing here (/rerun/DM-23243).  For gen. 3, use /rerun/RC/w_2020_26/DM-*."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# The coordinates of a nice barred spiral galaxy in COSMOS\n",
```

Comment:
Related to the comment above, were these coordinates coming from somewhere?

Reply:
As I understand, @aemerywatkins chose this galaxy by hand - do you have a reference catalogue we can cite?

Reply:
I chose this galaxy by using the hscMap tool found here: https://hsc-release.mtk.nao.ac.jp/doc/index.php/tools-2/
zooming in to the COSMOS field, and then panning around until I saw a lovely galaxy.  So we could either cite COSMOS and that tool, or, doing a quick NED search, here are its various catalogue names: CGCG 036-024; CGCG 1000.4+0235; MCG +01-26-011; WISEA J100259.05+022034.9; 2MASX J10025907+0220345

Reply:
I think that just adding a few of these details to the text would be fine. I was just confused because above it seemed like you were grabbing parameters from somewhere.